source/extensions/network/dns_resolver/apple/apple_dns_impl.h

```diff
@@ -101,6 +101,12 @@ class AppleDnsResolverImpl : public DnsResolver, protected Logger::Loggable<Logg
     // Network::ActiveDnsQuery
     void cancel(Network::ActiveDnsQuery::CancelReason reason) override;
 
+    // Network::ActiveDnsQuery
+    void addTrace(uint8_t) override {}
+
```

Comment:
Forconsistency you can put these all in one block with no spaces and only have one //Network::ActiveDnsQuery comment